parachain/pallets/ethereum-beacon-light-client/src/lib.rs

```diff
@@ -0,0 +1,666 @@
+//! # Ethereum 2 Light Client Verifier
+//!
+//! This module implements the `Verifier` interface. Other modules should reference
+//! this module using the `Verifier` type and perform verification using `Verifier::verify`.
+#![allow(unused_variables)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+mod merklization;
+
+use codec::{Decode, Encode};
+use frame_support::{dispatch::DispatchResult, log, transactional, Deserialize, Serialize};
+use frame_system::ensure_signed;
+use scale_info::TypeInfo;
+use sp_core::H256;
+use sp_runtime::RuntimeDebug;
+use sp_std::prelude::*;
+use sp_core::hashing::sha2_256;
```

Comment:
Actually this works:
```suggestion
use sp_io::hashing::sha2_256;
```